rocketpy/prints/sensors_prints.py

```diff
@@ -0,0 +1,118 @@
+        print("Rotation Matrix:")
+        for row in self.sensor.rotation_matrix:
+            value = " ".join(f"{val:.2f}" for val in row)
+            value = [float(val) for val in value.split()]
+            self._print_aligned("", value)
+
+    def quantization(self):
+        """Prints the quantization of the sensor."""
+        print("\nQuantization of the Sensor:\n")
+        self._print_aligned(
+            "Measurement Range:",
+            f"{self.sensor.measurement_range[0]} to {self.sensor.measurement_range[1]} ({self.units})",
+        )
+        self._print_aligned("Resolution:", f"{self.sensor.resolution} {self.units}/LSB")
+
+    @abstractmethod
+    def noise(self):
+        """Prints the noise of the sensor."""
+        pass
+
+    def _general_noise(self):
+        """Prints the noise of the sensor."""
+        print("\nNoise of the Sensor:\n")
+        self._print_aligned(
+            "Noise Density:", f"{self.sensor.noise_density} {self.units}/√Hz"
+        )
+        self._print_aligned(
+            "Noise Variance:", f"{self.sensor.noise_variance} ({self.units})^2"
+        )
+        self._print_aligned(
+            "Random Walk Density:",
+            f"{self.sensor.random_walk_density} {self.units}/√Hz",
+        )
+        self._print_aligned(
+            "Random Walk Variance:",
+            f"{self.sensor.random_walk_variance} ({self.units})^2",
+        )
+        self._print_aligned(
+            "Constant Bias:", f"{self.sensor.constant_bias} {self.units}"
+        )
+        self._print_aligned(
+            "Operating Temperature:", f"{self.sensor.operating_temperature} °C"
+        )
+        self._print_aligned(
+            "Temperature Bias:", f"{self.sensor.temperature_bias} {self.units}/°C"
+        )
+        self._print_aligned(
+            "Temperature Scale Factor:", f"{self.sensor.temperature_scale_factor} %/°C"
+        )
+        self._print_aligned(
+            "Cross Axis Sensitivity:", f"{self.sensor.cross_axis_sensitivity} %"
+        )
+
+    def all(self):
+        """Prints all information of the sensor."""
+        self.identity()
+        self.orientation()
+        self.quantization()
+        self.noise()
+
+
+class _AccelerometerPrints(_SensorsPrints):
+    """Class that contains all accelerometer prints."""
+
+    def __init__(self, accelerometer):
+        """Initialize the class."""
+        super().__init__(accelerometer)
+
+    def noise(self):
+        """Prints the noise of the sensor."""
+        self._general_noise()
+
+
+class _GyroscopePrints(_SensorsPrints):
+    """Class that contains all gyroscope prints."""
+
+    def __init__(self, gyroscope):
+        """Initialize the class."""
+        super().__init__(gyroscope)
+
+    def noise(self):
+        """Prints the noise of the sensor."""
+        self._general_noise()
+        self._print_aligned(
+            "Acceleration Sensitivity:",
+            f"{self.sensor.acceleration_sensitivity} rad/s/g",
+        )
```

Comment:
Seems like a good idea, but I prefer to leave it like this for now and discuss this later whenever I add the `Barometer`. There will be 3 new sensor classes in that PR so it will be better to discuss it over there